scripts/bump-engines.ts

```diff
@@ -20,7 +20,7 @@ async function main() {
   }
   await run(
     path.join(__dirname, '..'),
-    `pnpm update -r @prisma/engines-version@${version} @prisma/engines@${version}`,
+    `pnpm update -r @prisma/engines-version@${version} @prisma/engines@${version} @prisma/fetch-engine@${version} @prisma/get-platform@${version}`,
```

Comment:
huh @williamluke4?

Reply:
Note also done by Pierre in the interactive transaction PR